rootfs-builder/rootfs.sh

```diff
@@ -307,17 +327,46 @@ build_rootfs_distro()
 
 	mkdir -p ${ROOTFS_DIR}
 
+	# need to detect rustc's version too?
 	detect_go_version ||
 		die "Could not detect the required Go version for AGENT_VERSION='${AGENT_VERSION:-master}'."
 
 	echo "Required Go version: $GO_VERSION"
 
+	detect_rust_version ||
+		die "Could not detect the required rust version for AGENT_VERSION='${AGENT_VERSION:-master}'."
+
+	echo "Required rust version: $RUST_VERSION"
+
+	detect_cmake_version ||
+		die "Could not detect the required cmake version for AGENT_VERSION='${AGENT_VERSION:-master}'."
```

Comment:
should/do we print out the CMAKE versions we found/required when we `die` ?

Reply:
I don't know the minimum version required by building grpc, sorry

Reply:
when we die, we don't find any version, as detct_go_version behaves